### PR TITLE
Release/3.12.4

### PR DESCRIPTION
**Pull request by taquitos**
🐐  🥘  🍴       💳

### CODE REVIEW

**Review by aboedo — CHANGES_REQUESTED** [review]
we're missing the update to this line https://github.com/RevenueCat/purchases-ios/blob/RCv3_hotfix/Purchases/Misc/RCSystemInfo.m#L52, as well as jazzy, and the podspecs. 

the easiest way to get them all is to use the steps outlined in https://github.com/RevenueCat/purchases-ios/blob/main/RELEASING.md, with the `fastlane bump_and_update_changelog` command (or just `bump` now since you already have the changelog)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 3.12.4
+- Updated getOfferings call to be performed serially instead of concurrently.
```
Comment (aboedo) [inline CHANGELOG.md:2]: nitpick: maybe we could add backticks to `getOfferings`

**Comment by taquitos** [comment]
@aboedo what is crazy is that I ran that command and it didn't fail.

**Review by vegaro — APPROVED** [review]
Looks good now

**Review by aboedo — APPROVED** [review]
🚢 once tests pass

File: CHANGELOG.md (hunk 1 above)
Comment (taquitos) [reply CHANGELOG.md]: Good idea, done!

**Pull request merged**